pyproject.toml

```diff
@@ -71,9 +72,14 @@ multi_line_output = 3
 ignore_whitespace = true
 
 [[tool.mypy.overrides]]
-module = ["ape.*", "twisted.*"]
+module = ["ape.*", "twisted.*", "yaml.*"]
```

Comment:
Why this `yaml.*` thing?

Reply:
mypy complains yaml. stub is missing.

Reply:
> mypy complains yaml. stub is missing.

How does the error look like exactly? Aren't there typing stubs for yaml?

Reply:
beamer/tests/agent/multiple_processors/test_multiple_event_processors.py:13: error: Library stubs not installed for "yaml"  [import]
beamer/tests/agent/multiple_processors/test_multiple_event_processors.py:15: error: Library stubs not installed for "yaml.loader"  [import]
beamer/tests/agent/multiple_processors/test_multiple_event_processors.py:15: note: Hint: "python3 -m pip install types-PyYAML"
beamer/tests/agent/multiple_processors/test_multiple_event_processors.py:15: note: (or run "mypy --install-types" to install all missing stub packages)
beamer/tests/agent/multiple_processors/test_multiple_event_processors.py:15: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports

Reply:
So why don't we install the required typing packages? `types-PyYAML`?

Reply:
yes, now it is done.